build-files/config.json

```diff
@@ -0,0 +1,83 @@
+{
+  "GOOGLE_OAUTH_CLIENT_ID": "",
+  "GOOGLE_OAUTH_CLIENT_SECRET": "",
+
+  "BLOCK_ENCRYPTION_KEY": "",
+  "SECRET_ENCRYPTION_KEY": "",
+
+  "NYLAS_ADMIN_PASSWORD": "bbb4a88d392370c66a51512c4403353a386653f341bebdeaa030d289d8889cdc:e4e3670c6e0f4174b2a6acc2a33acf69",
```

Comment:
this password is for local development only. It's overwritten for production environment

Reply:
I think it's nice to leave the same comment in the code.